vignettes/articles/connect_hub.Rmd

```diff
@@ -9,7 +9,21 @@ knitr::opts_chunk$set(
 )
 ```
 
-An important function of `hubUtils` is allowing for the connection to data in the `model-output` directory to facilitate interrogation, filtering and extraction of Hub data.
+An important function of `hubUtils` is allowing for the connection to data in the `model-output` directory to facilitate extraction, filtering, querying, exploring, and analyzing of Hub data.
+
+# Structure of hubverse datasets
+
+All data returned from connecting to and querying hubs can be read or validated as a `model_out_tbl` which is an S3 class defined by the `hubUtils` package. A `model_out_tbl` is a long-form `data.frame` designed to conform to the [hubverse data specifications for model output data](https://hubdocs.readthedocs.io/en/latest/format/model-output.html). In short, the columns of a valid `model_out_tbl` containing model output data from a hub are:
+
+ - `model_id`: this is the unique character identifier of a model.
```

Comment:
We could mention here that `model_id` == `<team_abbr>_<model_abbr>`?

Reply:
That is true in some cases, but aren't there other cases where the `model_id` is just a single field identifier? I have lost track of where we ended up on this conversation.